sormas-api/src/main/java/de/symeda/sormas/api/symptoms/SymptomsHelper.java

```diff
@@ -195,7 +196,13 @@ public static void updateIsSymptomatic(SymptomsDto dto) {
 			throw new RuntimeException(e);
 		}
 
-		dto.setSymptomatic(false);
+		if (dto.getSymptomatic() == YesNoUnknown.UNKNOWN) {
```

Comment:
* See above concerning possible NullPointerExceptions
* You can write this way more efficiently like this:
```
if (!YesNoUnknown.UNKNOWN.equals(dto.getSymptomatic()) && !YesNoUnknown.NO.equals(dto.getSymptomatic())) {
  dto.setSymptomatic(null);
}
```